browser/brave_rewards/android/brave_rewards_native_worker.cc

```diff
@@ -71,6 +69,12 @@ void BraveRewardsNativeWorker::Destroy(JNIEnv* env) {
   delete this;
 }
 
+void BraveRewardsNativeWorker::CreateRewardsWallet(JNIEnv* env) {
+  if (brave_rewards_service_) {
+    brave_rewards_service_->CreateRewardsWallet(base::DoNothing());
```

Comment:
Should we have a callback here ?

Reply:
Good question. I don't think that the current Android implementation needs it (since it doesn't try to do any UI based on the result), but we may need to add callback support for the 2.5 UI. Do you think it would be OK to leave this without the callback in this PR?